packages/picasso.js/src/core/math/intersection.js

```diff
@@ -17,3 +17,25 @@ export function isCollinear(p1, p2, p3, e = EPSILON) {
   const t = Math.abs((p1.x * (p2.y - p3.y)) + (p2.x * (p3.y - p1.y)) + (p3.x * (p1.y - p2.y)));
   return t < e;
 }
+
+/**
+ * Check if the pie slice label is located entirely within it's parent rect
+ * @param {rect} childRect - A rect describing the position(x,y), width and height of label
+ * @param {rect} parentRect - A rect describing the position(x,y), width and height of parent rect
+ * @ignore
+ */
+export function rectContainsRect(childRect, parentRect) {
```

Comment:
Not sure what you are trying to achieve here? parentRect doesn't contain any x or y